browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -102,6 +102,8 @@ class NavigatorPanel extends SidebarBase {
 			closeNavWrapper,
 		);
 		closeNavButton.setAttribute('aria-label', _('Close Navigation'));
+		// on Intialization of sidebar make sure we set Navigator flag to false for safe check against cache
+		this.map.uiManager.setDocTypePref('ShowNavigator', false);
```

Comment:
best would be to do a class which will initialize all the values in single place, called from uiManager possibly on first initialize  of the UI

Reply:
`initializeBasicUI`

Reply:
to do later